plugins/generic/announcementFeed/AnnouncementFeedGatewayPlugin.php

```diff
@@ -161,6 +162,7 @@ public function fetch($args, $request)
             'dateUpdated' => $dateUpdated,
             'announcements' => $announcements,
             'journal' => $journal,
+            'language' => str_replace(['_', '@'], '-', Locale::getLocale()),
```

Comment:
I've seen this in other places, I think it should be a method somewhere ` toBcp47()`?!

Reply:
s. https://github.com/pkp/pkp-lib/issues/10795